spec/support/shared_example_spec_helper_for_integer_key.rb

```diff
@@ -28,10 +28,17 @@
   context "when try to create many records" do
 
     it "records created" do
-      expect { subject.create_many([
-                                     { :name => 'Alex', :company_id => 2, :integer_field => 4 },
-                                     { :name => 'Aaron', :company_id => 3, :integer_field => 2 }])
-      }.not_to raise_error
+      expect do
+        alex = { name: 'Alex', company_id: 2, integer_field: 4 }
+        aaron = { name: 'Aaron', company_id: 3, integer_field: 2 }
+
+        if ActiveRecord::VERSION::MAJOR < 5
+          subject.create_many([alex, aaron])
```

Comment:
`#create_many` is defined by another gem called bulk_update_methods.

- https://rubygems.org/gems/bulk_data_methods

This gem is no longer working with Rails 5. That might be okay, since Jora's codebases don't seem to use `#create_many` and `#update_many`.